binr/rahash2/rahash2.c

```diff
@@ -219,9 +219,9 @@ static int do_hash(const char *file, const char *algo, RIO *io, int bsize, int r
 				oto = to;
 				f = from;
 				t = to;
-				for (j = f; j < t; j += bsize) {
-					int nsize = (j + bsize < fsize) ? bsize : (fsize - j);
-					r_io_pread (io, j, buf, bsize);
+				for (j=f; j<t; j+=bsize) {
```

Comment:
``j = f``
and
``j=f``

---
``j<t``
and
``j < t``

---
``j+=bsize``
and
``j += bsize``

https://github.com/radare/radare2/blob/master/CONTRIBUTING.md
>Use a space after most of the keyword and around operators.